chart/templates/config.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.customRunConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "llama-stack.fullname" . }}-run-config
```

Comment:
Consider renaming the file `run-config.yaml` to differentiate it if additional config files are added in the futre.